model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -33,19 +33,26 @@ import org.ossreviewtoolkit.model.utils.FindingsMatcher
 import org.ossreviewtoolkit.model.utils.prependPath
 import org.ossreviewtoolkit.spdx.SpdxSingleLicenseExpression
 import org.ossreviewtoolkit.utils.CopyrightStatementsProcessor
+import org.ossreviewtoolkit.utils.FileMatcher
```

Comment:
Commit message nit: Could you maybe give an example what this function is good for, i.e. name a concrete use-case?

Reply:
Done.